rbx_binary/src/text_deserializer.rs

```diff
@@ -84,6 +85,30 @@ fn decode_meta_chunk<R: Read>(mut reader: R) -> DecodedChunk {
     DecodedChunk::Meta { entries, remaining }
 }
 
+fn decode_sstr_chunk<R: Read>(mut reader: R) -> DecodedChunk {
+    let version = reader.read_le_u32().unwrap();
+    let num_entries = reader.read_le_u32().unwrap();
+    let mut entries = Vec::with_capacity(num_entries as usize);
+
+    for _ in 0..num_entries {
+        let mut hash = [0; 16];
+        reader.read_exact(&mut hash).unwrap();
+
+        let data = reader.read_binary_string().unwrap();
+
+        entries.push(SharedString::new(data));
```

Comment:
Similar to other code, we can compress these statements a bit since they aren't very complicated.
```suggestion
        let data = reader.read_binary_string().unwrap();
        entries.push(SharedString::new(data));
```